### PR TITLE
KAS-4386: Refactor `Agenda::Agendaitem::AgendaitemDecision`

### PR DESCRIPTION
**Pull request by sergiofenoll**
https://kanselarij.atlassian.net/browse/KAS-4386

The component is split into two, one for legacy agendas and one for the new "digital" agendas.

Down the line we should further split up the digital component, as all the bits for editing the piece parts can be de-duplicated and placed in separate parametrized components. I chose not to make this change here to keep this PR simple. 

The current refactor is purely structural, code was moved but the logic remains the same. I didn't want to introduce possible logic changes (caused by refactoring the piece part logic) here. If this gets merged, we can start looking at moving off more stuff, without having to be worried about this refactor having broken stuff.

### CODE REVIEW

File: app/components/agenda/agendaitem/decision.hbs
Comment (kobemertens) [inline app/components/agenda/agendaitem/decision.hbs:3]: Love the param annotations!

**Review by kobemertens — APPROVED** [review]
Tested locally, works great.

**Review by ValenberghsSven — APPROVED** [review]
Looks good
New code was mostly auto merged in digital, which makes it easier
I did not apply all new code to legacy (like hiding newsitem based on decision result)
Because legacy is not active edit view. Many of the automations may have to go to prevent any side effects should users attempt to edit legacy

Fixed 1 typo that needed to be fixed in both templates but auto merge only fixed it in digital

Wait for jenkins, but should be ok to merge if nothing strange comes out of that.

**Pull request merged**
